app/views/comments/_form.html.erb

```diff
@@ -12,8 +12,15 @@
   <% end %>
 
   <div class="form-group dropzone">
-    <textarea style="border: 1px solid #bbb;border-bottom-left-radius: 0;border-bottom-right-radius: 0;border-bottom: 0;padding: 10px;" onFocus="editing=true" name="body" class="form-control" id="text-input" rows="6" cols="40" placeholder="<%= placeholder %>"><%= params[:body] %></textarea>
-
+    <% 
+      body = params[:body].nil?
+      # Look for comment templates
+      if params[:body].empty? && @node && @node.has_power_tag('comment-template')
```

Comment:
Oh I'm sorry, it looks like "empty?" is still here, actually the .nil? you added on line 16 ought to be here, replacing .empty?, do you think you could make that last change? I think thatll resolve the errors here!

My apologies for missing your response, there's been a lot of activity on the project the past week!!!